output.cpp

```diff
@@ -135,17 +167,24 @@ void fb_output(input_t *input) {
   }
   
   fprintf(f,"#");
-  fprintf(f,"Subpopulation order: %s", input->reference_subpops[0]);
+  fprintf(f,"reference_panel_population:\t%s", input->reference_subpops[0]);
   for(int i=1; i < input->n_subpops; i++) {
     fprintf(f,"\t%s", input->reference_subpops[i]);
   }
   fprintf(f,"\n");
-  fprintf(f,"#chm\tpos\tgpos\tsnp idx");
+  fprintf(f,"chromosome\tphysical_position\tgenetic_position\tgenetic_marker_index");
   for(int j=0; j < input->n_samples; j++) {
     sample_t *sample = input->samples + j;
     if (sample->apriori_subpop != -1 || sample->s_sample == 1) continue;
 
-    fprintf(f,"\t%s.0\t%s.1", sample->sample_id, sample->sample_id);
+    for(int k=0; k < input->n_subpops; k++) {
+      fprintf(f, "\t%s:::hap1:::%s:::probability", sample->sample_id, input->reference_subpops[k]);
+    }
+    for(int k=0; k < input->n_subpops; k++) {
+      fprintf(f, "\t%s:::hap2:::%s:::probability", sample->sample_id, input->reference_subpops[k]);
+    }
+
+    // fprintf(f,"\t%s.0\t%s.1", sample->sample_id, sample->sample_id);
```

Comment:
oh, and this is bad: If you're going to leave a line commented out like this, definitely put a comment next to it explaining why. 
Commented out code without any explanation is a maintenance nightmare